pom.xml

```diff
@@ -301,18 +301,24 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-cps</artifactId>
-            <version>2.29</version>
+            <version>2.32</version>
         </dependency>
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-api</artifactId>
-            <version>2.13</version>
+            <version>2.16</version>
         </dependency>
+        
+        <!-- 
+            workflow-job >= 2.12 requires a Jenkins core >= 2.60.
+            Do not upgrade this dependency without a discussion on jenkinsci-ux@googlegroups.com
+        -->
```

Comment:
As tested with experimental update center, there might be beta version of other dependencies out there that might depend on it resulting in to blueocean install errors - specially if blueocean is installed on latest LTS (2.46.3) without installing required plugins.